stub-idp/src/main/java/uk/gov/ida/stub/idp/services/AuthnRequestReceiverService.java

```diff
@@ -16,13 +16,15 @@
 import uk.gov.ida.stub.idp.repositories.IdpSessionRepository;
 
 import javax.inject.Inject;
+import javax.swing.text.html.Option;
```

Comment:
Where is this used?

Reply:
Its not, think must have been introduced as part of an earlier Intellisense error and I hadn't optimised imports since.

Reply:
Removed